custom_components/alexa_media/media_player.py

```diff
@@ -1384,6 +1386,18 @@ async def async_play_tts_cloud_say(self, public_url, media_id, **kwargs):
             file_name = media.url[media.url.rindex("/") : media.url.rindex(".")]
             media_id = async_process_play_media_url(self.hass, media.url)
 
+        if media_type == "music":
+            # Log and notify for Amazon restriction on streaming music
+            _LOGGER.warning(
+                "Sorry folks! Amazon doesn't allow streaming music like this. "
```

Comment:
Probably should just save the message as a string that we reference by variable name instead of duplicating.